ports/nrf/Makefile

```diff
@@ -213,7 +213,7 @@ FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py')
 FROZEN_MPY_MPY_FILES := $(addprefix $(BUILD)/,$(FROZEN_MPY_PY_FILES:.py=.mpy))
 
 OBJ += $(PY_O) $(SUPERVISOR_O) $(addprefix $(BUILD)/, $(SRC_C:.c=.o))
-OBJ += $(BUILD)/pins_gen.o
+OBJ += $(BUILD)/pins_gen.o supervisor/cpu.o
```

Comment:
Adding a SRC_S variable and then using addprefix to change to .o would probably be cleaner :)

Reply:
Done. Thanks for the comment